src/shared/components/StudyList/StudyList.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import {TypeOfCancer as CancerType, CancerStudy} from "../../api/generated/CBioPortalAPI";
+import * as styles_any from './styles.module.scss';
+import classNames from "../../lib/classNames";
+import FontAwesome from "react-fontawesome";
+import LabeledCheckbox from "../labeledCheckbox/LabeledCheckbox";
+import {observer} from "mobx-react";
+import {getStudySummaryUrl, getPubMedUrl} from "../../api/urls";
+import {QueryStoreComponent} from "../query/QueryStore";
+
+const styles = {
+	...styles_any as {

```

Comment:
Because I'm adding properties below, I'm using the splat operator to copy the properties from styles_any, and I'm assigning strong typings.